apps/web/pages/api/auth/[...nextauth].ts

```diff
@@ -69,23 +75,49 @@ export default NextAuth({
         };
       },
     }),
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID!,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET!,
+    }),
   ],
   callbacks: {
     async jwt({ token, user, account }) {
       return {
         ...token,
       };
     },
-    async session({ session, token }) {
+    async session({ session, token, user }) {
       const documensoSession: Session = {
         ...session,
         user: {
           ...session.user,
+          id: Number(token.sub),
```

Comment:
Any specific reason to have id as a number in `session` on the client?

Reply:
From the schema, the id is stored as a number, hence we are maintaining the same type across the application.